### ISSUE TITLE
KeyError: `REPL not found`

### ISSUE THREAD

**Issue by lgoettgens** [LilithHafner/BasicAutoloads.jl#7]
Starting with julia 1.11.0, every time I use one of the macros backed by my startup.jl, I get more or less the following error:
```julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.11.0 (2024-10-07)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> @benchmark
┌ Info: Failed to run `using BenchmarkTools`
└   exception = KeyError: key Base.PkgId(Base.UUID("3fa0cd96-eef1-5676-8a61-b3b8758bbffb"), "REPL") not found
ERROR: LoadError: UndefVarError: `@benchmark` not defined in `Main`
Suggestion: check for spelling errors or missing imports.
in expression starting at REPL[1]:1
```
Do you by chance know why this happens or what to do to fix it?
My `startup.jl` is a copy from this repositories readme, with some more things added for e.g. Jet and Cthulhu.

**Comment by LilithHafner** [comment]
Good catch! 

This is because in Julia 1.11, REPL is not in `Base.loaded_modules`, even when the REPL is displayed to the user. I fixed this by getting a handle on the existing REPL module in a different way (inspecting `Base.active_repl_backend`).

**Comment by LilithHafner** [comment]
Should be fixed in BasicAutoloads 1.0.1, [just released](https://github.com/JuliaRegistries/General/pull/117163)
